.github/workflows/test_jax_wheels.yml

```diff
@@ -0,0 +1,66 @@
+name: Test JAX Wheels
+
+on:
+  workflow_dispatch:
+    inputs:
+      amdgpu_family:
+        required: true
+        type: string
+        default: "gfx94X-dcgpu"
+      test_runs_on:
+        required: true
+        type: string
+        default: "linux-mi325-1gpu-ossci-rocm"
+      cloudfront_url:
+        description: CloudFront URL pointing to Python index
+        required: true
+        type: string
+        default: "https://d2awnip2yjpvqn.cloudfront.net/v2"
+      python_version:
+        required: true
+        type: string
+        default: "3.12"
```

Comment:
You may also want a `jax_version` like how `torch_version` is used here, so you have control over which version is installed and tested: https://github.com/ROCm/TheRock/blob/90299bcdb49d7a7788ee74e666c5e62c487192eb/.github/workflows/test_pytorch_wheels.yml#L23-L25

(that can be added once you have this stub workflow though)